utils/notification_service.py

```diff
@@ -1220,7 +1227,8 @@ def prepare_reports(title, header, reports, to_truncate=True):
         os.makedirs(os.path.join(os.getcwd(), f"ci_results_{job_name}"))
 
     target_workflow = "huggingface/transformers/.github/workflows/self-scheduled-caller.yml@refs/heads/main"
-    is_scheduled_ci_run = os.environ.get("CI_WORKFLOW_REF") == target_workflow
+    amd_target_workflow = "huggingface/transformers/.github/workflows/self-scheduled-amd-caller.yml@refs/heads/main"
+    is_scheduled_ci_run = os.environ.get("CI_WORKFLOW_REF") in [target_workflow, amd_target_workflow]
```

Comment:
I am a bit uncertain here

> CI_WORKFLOW_REF: ${{ github.workflow_ref }}

when it comes to AMD CI (called from `hf-workflows` ), what `github.workflow_ref ` would be ?

Reply:
oh my bad.

The triggered workflow run would be `.github/workflows/self-scheduled-amd-caller.yml`, and `hf-workflows` ones are being called